fuel-core-interfaces/src/utils.rs

```diff
@@ -0,0 +1,18 @@
+pub mod signer {
```

Comment:
This change seems pointless and unrelated to storage updates. I don't think we should be randomly stuffing things into `util` modules